<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>553</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="3">
         <widget class="QPushButton" name="right">
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>right</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="up">
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>up</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="openFile">
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Open file</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="checkBox">
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>24</height>
           </size>
          </property>
          <property name="font">
           <font>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>figure</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="left">
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>left</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="down">
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>down</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox">
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="settings">
          <property name="text">
           <string>settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="get_closer">
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Get closer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="move_it_away">
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Move it away</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="fileName">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countOfVertices">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Vertices: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countOfSurface">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Surfaces:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
